examples/native-fungible/tests/transfers.rs

```diff
@@ -167,7 +167,11 @@ async fn emptied_account_disappears_from_queries() {
 
     recipient_chain
         .add_block(|block| {
-            block.with_native_token_transfer(Some(owner), Recipient::Burn, transfer_amount);
+            block.with_native_token_transfer(
+                AccountOwner::User(owner),
+                Recipient::Burn,
```

Comment:
We can probably kill this `Burn` thing by the way and simplify `Recipient` now.

Reply:
https://github.com/linera-io/linera-protocol/issues/3580